src/protocol/error.js

```diff
@@ -87,12 +90,14 @@ const errorCodes = [
     type: 'NETWORK_EXCEPTION',
     code: 13,
     retriable: true,
+    transient: true,
     message: 'The server disconnected before a response was received',
   },
   {
     type: 'GROUP_LOAD_IN_PROGRESS',
     code: 14,
     retriable: true,
+    transient: true,
```

Comment:
I was a bit conflicted whether to introduce property _transient_.  For example, some errors (e.g. REBALANCE_IN_PROGRESS, REASSIGNMENT_IN_PROGRESS) seem to be transient but not classified as _retriable_
Thoughts?

Reply:
Hmm, well, `REBALANCE_IN_PROGRESS`, for example, isn't a retriable error because retrying won't ever solve it (you need to re-join). So you're right that we need something else to explain that they are errors that KafkaJS will try to take care of itself. I don't think the name `transient` is quite right on the protocol level though (i.e. `protocol/errors`), since on the protocol level these errors aren't transient (unless they're retriable). They are only transient because higher-level code tries to handle the errors.

I'm not sure if the solution is to introduce something a bit higher up that marks the errors as transient (or sets a log level per error or something), or simply renaming the property from `transient` to `handled`.

@tulios Do you have any thoughts on this?

Reply:
Extremely late for this, but I like `handled` more as this is exactly the case. We can even make it more explicit by calling it `handledByClient`, WDYT @Nevon?

Reply:
I think `handled` is a good name.